src/check_msg.c

```diff
@@ -24,6 +24,7 @@
 #include <stdlib.h>
 #include <fcntl.h>
 #include <stdio.h>
+#include <unistd.h>
```

Comment:
This header is not available if in the Windows environment. Because of this, the MSVC and Visual Studio build fails: https://ci.appveyor.com/project/brarcher46583/check/build/1.0.467.